src/components/TextInput/TextInput.tsx

```diff
@@ -54,16 +56,22 @@ export const TextInput = React.forwardRef<HTMLSpanElement, TextInputProps>(funct
         style,
         className,
         qa,
+        innerLabelProps,
```

Comment:
Not sure if we want to allow any customization here

Reply:
by default `max-width` is set at 50% parent container. but sometimes at use place we might want to set it in fixed `px` or different `%`.